### PR TITLE
[C-5556] adds support for audiences in courier

### PR DESCRIPTION
**Pull request by suhasdeshpande**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trycourier/courier",
-  "version": "3.9.0",
+  "version": "3.10.0",
```
Comment (tk26) [inline package.json:3]: @suhasdeshpande did we miss a version? changelog says 3.11

Comment (suhasdeshpande) [reply package.json:3]: maybe @lednax?? This should be `3.11.0` I saw @lednax merged something

Comment (suhasdeshpande) [reply package.json:3]:
```suggestion
  "version": "3.11.0",
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -221,7 +226,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## v1.0.1 - 2019-07-12
 
-[unreleased]: https://github.com/trycourier/courier-node/compare/v3.9.0...HEAD
+[unreleased]: https://github.com/trycourier/courier-node/compare/v3.11.0...HEAD
+[v3.10.0]: https://github.com/trycourier/courier-node/compare/v3.10.0...v3.11.0
```
Comment (tk26) [inline CHANGELOG.md:230]:
```suggestion
[v3.11.0]: https://github.com/trycourier/courier-node/compare/v3.10.0...v3.11.0
```

**Review by tk26 — APPROVED** [review]

**Pull request merged**